README.md

```diff
@@ -1,2 +1,26 @@
 # toolhunt
-A web application for editing Toolhub records in a fun and easy way
+# Description
+A web application for editing Toolhub records in a fun and easy way.
+It allows users to view and add missing fields for tools in Toolhub.
+This is an Outreachy Internship project.
+
+## Issue Tracker Link 
+ Click [View Site](https://phabricator.wikimedia.org/project/board/6283/)  to visit the site
```

Comment:
I'd be more explicit about us using Phabricator for issue tracking (right now it's hidden in the link), and also advising against using GitHub's own issue tracker for bugs, etc. (There may actually be a way to disable GH issues from the settings – I'll investigate).